KPI/README.md

```diff
@@ -0,0 +1,111 @@
+# KPI Python Computation
+
+In this directory, we will store code, high-level tasks, discussions, and material, related to the research activity conducted together with [AISM - Associazione Italiana Sclerosi Multipla](https://www.aism.it/), under the superfivion of Gianpaolo Brichetto.  This research is developed within the framework of [iFeel](https://github.com/ami-iit/component_ifeel) project and aims to analyze/validate our technology in the rehabilitation context of multiple sclerosis patients.
+
+## Guide to use the code
+This project has been updated to make the code more accessible and user-friendly. Below are the steps to execute the code for KPI calculation in Python.
```

Comment:
Imagine an external reader that reads the README for the first time. I do not think that he/she will have enough context to understand what "This project has been updated to make the code more accessible and user-friendly. " sentence means in this context.